app/views/schools/advice/electricity_out_of_hours/_by_day.html.erb

```diff
@@ -0,0 +1,24 @@
+<%= render 'schools/advice/section_title',
+           section_id: 'usage_by_day_of_week',
+           section_title:
+           t('advice_pages.electricity_out_of_hours.analysis.usage_by_day_of_week.title') %>
+<%= component 'chart', chart_type: :electricity_by_day_of_week_tolerant, school: school do |c| %>
+  <% c.with_title do %>
+    <%# i18n-tasks-use t('advice_pages.electricity_out_of_hours.analysis.usage_by_day_of_week.electricity_by_day_of_week_tolerant_chart.title') %>
+    <%= t('advice_pages.electricity_out_of_hours.'\
+          'analysis.usage_by_day_of_week.electricity_by_day_of_week_tolerant_chart.title') %>
```

Comment:
Splitting lines to keep ERB happy, but then having to add hints without split strings to keep i18n-tasks happy. Even if we factored out a prefix, we'd still need the hints. Maybe we should just allow longer lines? Or longer lines in ERB files?